ui.apps/src/main/content/jcr_root/apps/hclecomm/config.publish/com.hcl.ecomm.core.services.impl.ProductServiceImpl.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@DeepaliAgrwl : why do we have a separate runmode for publish ?

Reply:
Removed